samples/21.core-bot-app-insights/src/main/resources/application.properties

```diff
@@ -0,0 +1,7 @@
+MicrosoftAppId=
+MicrosoftAppPassword=
+LuisAppId=
+LuisAPIKey=
+LuisAPIHostName=
+ApplicationInsightsInstrumentationKey=
```

Comment:
Are we sure about [this](https://github.com/microsoft/BotBuilder-Samples/blob/main/samples/csharp_dotnetcore/21.corebot-app-insights/appsettings.json#L7)? Have you found another example?

Reply:
nope. Looked but didn't find. I was precisely in doubt about this. In [C#](https://github.com/microsoft/BotBuilder-Samples/blob/main/samples/csharp_dotnetcore/21.corebot-app-insights/appsettings.json#L7) the property is a structure with only one value so I bet on making it one value in Java. I expect we would find out what the correct implementation is when we start working on the rest of the bot

Reply:
What about `ApplicationInsights.InstrumentationKey`? I'm following the approach of this [sample](https://github.com/microsoft/botbuilder-java/blob/main/samples/81.skills-skilldialog/dialog-root-bot/src/main/resources/application.properties#L6)

Also, you are missing a comment, this is the way to can [add comments](https://github.com/microsoft/botbuilder-java/blob/main/samples/81.skills-skilldialog/dialog-skill-bot/src/main/resources/application.properties#L8)

Reply:
Excellent ideas. Will change it